server/jetstream_cluster.go

```diff
@@ -2983,6 +2983,17 @@ func (js *jetStream) applyStreamEntries(mset *stream, ce *CommittedEntry, isReco
 					mset.clMu.Unlock()
 				}
 
+				if mset.inflightSubjects != nil {
+					mset.clMu.Lock()
+					n := mset.inflightSubjects[subject]
```

Comment:
We always tracking these?

Reply:
Or only for discard new per subject?

Reply:
Only for discard new + discard new per subject for wq or interest streams (see line#7932)